src/neo-vm/ExecutionEngine.cs

```diff
@@ -299,6 +299,13 @@ private void ExecuteInstruction()
                         ExecuteCall(x.Position);
                         break;
                     }
+                case OpCode.CALLT:
+                    {
+                        ExecutionContext context_current = CurrentContext;
+                        IToken token = LoadToken(instruction.TokenU16, out ExecutionContext context_loaded);
+                        context_current.EvaluationStack.MoveTo(context_loaded.EvaluationStack, token.ParametersCount);
```

Comment:
We should copy `ReturnValuesCount` when unload the context?

Reply:
Done.